Marlin/src/module/motion.h

```diff
@@ -289,14 +289,14 @@ void homeaxis(const AxisEnum axis);
 
   // Return true if the given position is within the machine bounds.
   inline bool position_is_reachable(const float &rx, const float &ry) {
-    if (!WITHIN(ry, Y_MIN_POS - slop, Y_MAX_POS + slop)) return false;
+    if (!WITHIN(ry, LOGICAL_Y_POSITION(Y_MIN_POS - slop), RAW_Y_POSITION(Y_MAX_POS + slop))) return false;
```

Comment:
This is definitely incorrect. `position_is_reachable` takes raw (native) coordinates and checks them against these hard limits, which are also expressed in native coordinates. These values should not be treated in any other way here in this core and fundamental method.